develop/learning-paths/migration/articles/01-intro/01-from-a-monolith-to-a-modular-architecture.markdown

```diff
@@ -0,0 +1,124 @@
+# From a Monolith to a Modular Architecture
+
+Since Liferay Portal's beginning it's been an amazing application full of all
```

Comment:
This is a minor thing, but I tend to think that our documentation should be objective and avoid terms such as "amazing" or even "full of all kinds" which could be seen as an exaggeration from our side. For anyone thinking that way the documentation may loose credibility after reading this.